### ISSUE TITLE
Random question: Would this be suitable for real-time state estimation tasks?

### ISSUE THREAD

**Issue by zbeekman** [scivision/pyoptflow#4]
Basically, I'm wondering if you could combine optical flow data from a camera with other sensor data using something like a KF or UKF. The application would be model aircraft, drones or stabilized model rockets. So the optic flow should be due primarily to rotations, translations and perspective effects. The camera would be aligned with one of the vehicles axes and could look at just rotation about that axis, for example.

Basically I'm wondering how fast it is, and if there is a way that you could feed it live data from a camera.

Obviously, I could hunt through the code, so answers that are equally as lazy as my question are welcome, haha!

**Comment by scivision** [comment]
I haven't compared it against modern "single camera" techniques. Probably want to use a C implementation.
